shogun/cpp/wscript

```diff
@@ -0,0 +1,20 @@
+def options(opt):
+    opt.load('compiler_cxx')
+
+
+def configure(conf):
+  conf.env.CXXFLAGS += ['-O2', '-Wall', '-g', '-pipe']
+  #conf.env.CXXFLAGS += ['-std=c++11']
+  conf.load('compiler_cxx')
+  #conf.check_cfg(package = 'pficommon', args = '--cflags --libs')
+  conf.check_cxx(lib = 'msgpack')
+  conf.check_cxx(lib = 'jubatus_mpio')
+  conf.check_cxx(lib = 'jubatus_msgpack-rpc')
+
+
+def build(bld):
+  bld.program(
+    source = 'shogun.cpp',
+    target = 'shogun',
+    use = 'PFICOMMON MSGPACK JUBATUS_MPIO JUBATUS_MSGPACK-RPC',
```

Comment:
PFICOMMON is unused in this program, so please remove `PFICOMMON` from `use` list.